verify_rpms/cmd_runner.py

```diff
@@ -0,0 +1,23 @@
+from subprocess import run, CompletedProcess, CalledProcessError
+from typing import List
+from verify_rpms.exceptions import CmdError
+
+
+class CmdRunner:
+
+    @staticmethod
+    def run_cmd(args: List[str]) -> CompletedProcess:
```

Comment:
What's the idea behind adding this instead of using subprocess.run directly?

Reply:
We will use it more than once to run commands, and instead of defining it for every action, I am defining it once with the attributes I need and with the exception handling

later it will be injected to the classes that will run the commands

Reply:
IMO the whole thing can be replaced with a `partial` call, it seems to be on overkill for me. Also In practice I wouldn't even want the partial call because I'd rather see the arguments passed to `run` explicitly in the place the command is run.